conference_schema.yml

```diff
@@ -0,0 +1,44 @@
+# Schema for conference data
+
+type:       map
+mapping:
+  "name":
+    type:      str
+    required:  True
+  "website":
+    type:      str
+    required:  True
+    pattern:   "https?://"
+  "location":
+    type:      str
+    required:  True
+  "status":
+    type:      str
+    required:  False
+  "date_start":
+    type:      date
+    required:  True
+    format:    "YYYY-MM-DD"
+  "date_end":
+    type:      date
+    required:  True
+    format:    "YYYY-MM-DD"
+  "online":
+    type:      bool
+    required:  False
+  "cfp":
+    type:      map
+    required:  False
+    mapping:
+      "start":
+        type:      date
+        required:  True
+        format:    "YYYY-MM-DD"
+      "end":
+        type:      date
+        required:  True
+        format:    "YYYY-MM-DD"
+      "site":
+        type:      str
+        required:  False
```

Comment:
The template does not handle the case where this isn't available. If we want to make it optional we should not emit an anchor tag.

Reply:
We fall back to the `website` value if the CfP site is missing.

https://github.com/AndroidStudyGroup/conferences/blob/01ed2a5bed7ce13cc2fe75d2cb24343f05ea7b5b/index.html#L29

Reply:
Perfect